docs/source/getting-started/content/proc_starting-codeready-containers-with-proxy.adoc

```diff
@@ -0,0 +1,20 @@
+[id="starting-codeready-containers-with-proxy_{context}"]
+= Starting {prod} behind proxy
+
+Proxy can be defined as part of environment variable or with `crc config set` command option as following:
+
+```
+$ crc config set http-proxy http://example.proxy.com:<port>
+$ crc config set https-proxy http://example.proxy.com:<port>
+$ crc config set no-proxy '<multiple-no-proxy-entries-with-comma-separated>'
+```
+
+Once proxy is set by environment variable or by config, `crc` commands will able to use it.
+
+
+[NOTE]
+====
+- If both config and env are set for proxy, the values set in crc config have priority over the ones set in the environment.
```

Comment:
We should avoid using "if" in documentation. We can simply say that configuration values take precedence over environment variables.

To be perfectly honest, though, this feels backwards to me -- should environment variables take precedence over configuration? (That's not a question for docs, but I thought it was worth asking.)

Reply:
> To be perfectly honest, though, this feels backwards to me -- shouldn't environment variables take precedence over configuration? (That's not a question for docs, but I thought it was worth asking.)

@robin-owen no ENV variable shouldn't take precedence over application configuration/variable, ENV variables are always there in case it is not defined it the application the use it so it is correct imo.